lib/Service/LangRopeService.php

```diff
@@ -47,7 +49,7 @@ private function getExApp() {
 
 	/**
 	 * @param string $userId
-	 * @param Source[] $sourceNames
```

Comment:
thanks for that change, makes things much easier :)

Reply:
the endpoint was already supposed to use source names, sorry about the mislabel of the endpoint.